homeassistant/components/deconz/.translations/en.json

```diff
@@ -51,5 +51,11 @@
                 "description": "Configure visibility of deCONZ device types"
             }
         }
+    },
+    "device_automation": {
+        "trigger_type": {
+            "turn_on": "{name} turn on button pressed",
```

Comment:
It took me 5 readings before I understood this. Can we put quotes around "turn on"?

Also, let's make the key not just `turn_on`, but instead be more precise, like `remote_turn_on`

Reply:
Actually wondering if we can come up with a better sentence.

Reply:
CC @SeanPM5